src/unit_tests/os_crypto/sha512/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
```

Comment:
Considering [this](https://github.com/wazuh/wazuh/pull/11780/files) PR, I'd just use the year 2015 in new files

Reply:
Done